src/solvers/dgsem_t8code/containers.jl

```diff
@@ -0,0 +1,329 @@
+mutable struct T8codeElementContainer{NDIMS, RealT<:Real, uEltype<:Real, NDIMSP1, NDIMSP2, NDIMSP3} <: AbstractContainer
```

Comment:
You mentioned that you basically copied everything here from the `P4estMesh`. Could you please explain why you did so and why you cannot re-use the other container structures?

Reply:
I said this wrong. `T8codeMesh` and `P4estMesh` are very similar but differ in some aspects. What I meant where the interface, mortar and boundary containers. They are identical. Sorry about the mix-up.

Reply:
Okay. Could you please explain why you copied the containers and why you cannot re-use the other container structures?

Reply:
I need this for method dispatch. Even though the cache containers are identical to ones for `P4estMesh` the routines to fill them are specialized for `T8codeMesh`.

Reply:
Could you please point me to the place where you need this dispatch?

Reply:
Thanks :+1: